src/components/Contact/Chat.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { StyleSheet, Text, View, TouchableOpacity, Linking } from 'react-native';
+
+
+export default class Chat extends React.Component {
+  render() {
+    return (
+      <TouchableOpacity onPress={() => Linking.openURL('https://m.me/breakfasy')}>
+        <Text style={styles.chat}>
+          Chat with us on messenger!
+        </Text>
+      </TouchableOpacity>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  chat: {
+    color: 'white',
+    fontSize: 20,
+    textAlign: 'center',
+    textDecorationLine: 'underline',
+    verticalAlign: 'middle',
+    top: '36%',
+  },
+});
```

Comment:
Contact/Chat.js: Good use of `TouchableOpacity onPress` and `Linking.openURL` to enable visitors to interact with fortissimo. Code is clean and well-organized for future additions/amendments.